setup.cfg

```diff
@@ -1,7 +1,73 @@
 [metadata]
-long_description = file: README.md
+name = pipenv
+author = Pipenv maintainer team
+author_email = distutils-sig@python.org
```

Comment:
We should figure out something to change this to at some point since that email list is gone.